test/management/rules-configs.tests.js

```diff
@@ -1,156 +1,125 @@
 var expect = require('chai').expect;
```

Comment:
Not sure why this file is so different, maybe precommit hook?

Reply:
Yeah. Not sure why it wasn't formatted before..